providers/fips/self_test_data.inc

```diff
@@ -386,6 +386,25 @@ static const ST_KAT_PARAM pbkdf2_params[] = {
     ST_KAT_PARAM_END()
 };
 
+static const char snmpkdf_digest[] = "SHA1";
```

Comment:
Maybe reference where this comes from in a comment..

Reply:
done